deploy/clowdapp.yaml

```diff
@@ -54,6 +54,8 @@ objects:
           env:
             - name: INSIGHTS_RESULTS_SMART_PROXY__SERVER__ADDRESS
               value: ":8000"
+            - name: INSIGHTS_RESULTS_SMART_PROXY__SERVER__API_DBG_PREFIX
```

Comment:
These lines will enable the debug endpoints, right? If this env var is deployed to prod, we will be serving the debug endpoint, which I don't think that it is what we want. Am I wrong?

Reply:
@joselsegura no, this only defines env. variable for prefix name. For enable/disable debug in prod there is variable INSIGHTS_RESULTS_SMART_PROXY_SERVER_DEBUG.

Reply:
Great, LGTM then